README.md

```diff
@@ -69,27 +66,31 @@ namespace netCLIConsoleApp
 ## Using dependency injection:
 
 ```cs
-using inausoft.netCLI;
-using Microsoft.Extensions.DependencyInjection;
-using System;
+...
 
 namespace ConsoleApp1
 {
     class Program
     {
-        static void Main(string[] args)
+        static int Main(string[] args)
         {
             var services = new ServiceCollection();
-            services.AddCLI(config => {
-                config.Map<MoveCommand, MoveCommandHandler>();
-            });
+            services.ConfigureCLFlow(mapping =>
+            {
+                mapping.Map<MoveCommand, MoveCommandHandler>()
+                       .Map<HelpCommand, HelpCommandHandler>();
 
-            var provider = services.BuildServiceProvider();
+            });
 
-            return provider.RunCLI(args);
+            using( var provider = services.BuildServiceProvider())
+            {
+                return CLFlow.Create().UseServiceProvider(provider)
+                                      .Run(args);
+            }
         }
     }
 }
+...
```

Comment:
Is this required? I hope that when I copy the code it just works.

Reply:
`...` indicates that there are some code missing - and that the example is only partial.